File: app/src/main/java/com/jermaine/awesomelibraries/api/ApiService.java
package com.jermaine.awesomelibraries.api;


import com.jermaine.awesomelibraries.api.response.Repo;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {
    @GET("/users/JakeWharton/repos")
    Call<List<Repo>> getRepos();
}

File: app/src/main/java/com/jermaine/awesomelibraries/api/ApiService.kt
package com.jermaine.awesomelibraries.api


import com.jermaine.awesomelibraries.datamodel.repo.Repo

import retrofit2.Call
import retrofit2.http.GET

interface ApiService {
    @get:GET("/users/jermainedilao/repos")
    val repos: Call<List<Repo>>
}